extension/src/test/suite/experiments/data/index.test.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from '../../util'
 import { dvcDemoPath, getTestWorkspaceFolder } from '../../../util'
 import { ExperimentsData } from '../../../../experiments/data'
-import * as Watcher from '../../../../fileSystem/watcher'
```

Comment:
[F] I think at some point this was stubbed somewhere in this suite but it is not anymore.